### PR TITLE
add support for line break mode

### PR DESCRIPTION
**Pull request by tettoffensive**
For https://github.com/Raizlabs/BonMot/issues/111

### CODE REVIEW

File: Pod/Classes/BONText.m [hunk 1]
```diff
@@ -311,6 +311,13 @@ - (BONStringDict *)attributes
         populateParagraphStyleIfNecessary();
         paragraphStyle.lineSpacing = self.lineSpacing;
     }
+    
+    // Line Break Mode
+    
+    if (self.lineBreakMode != NSLineBreakByWordWrapping) {
```
Comment (ZevEisenberg) [inline Pod/Classes/BONText.m]: I would prefer not to rely on the implementation detail that this property is initialized to `0`, which happens to equal `NSLineBreakByWordWrapping`. Please add that to `-init` of `BONText.m`.

**Comment by ZevEisenberg** [comment]
@tettoffensive thanks for your PR! If you'd be willing to address my one comment, then I'd happily test and merge this.

**Comment by tettoffensive** [comment]
@ZevEisenberg happy to address. I've added to the init

File: Example/Tests/BONTextAlignmentTestCase.m [hunk 2]
```diff
@@ -75,6 +75,7 @@ - (void)testParagraphStyle
                           .headIndent(2.34)
                           .tailIndent(3.45)
                           .lineHeightMultiple(3.14)
+                          .lineBreakMode(NSLineBreakByTruncatingTail)
```
Comment (ZevEisenberg) [inline Example/Tests/BONTextAlignmentTestCase.m]: Would you mind moving this down below `.lineSpacing(2.72)`? Line break mode is after line spacing everywhere else, and that consistency makes it very easy to validate that it's in all the places it should be.

**Comment by ZevEisenberg** [comment]
@tettoffensive just the one additional comment, and then I'm good to go. I double checked and tested it, and it looks great.

**Comment by tettoffensive** [comment]
@ZevEisenberg moved the line

**Pull request merged**